src/components/ActionButton/ActionButton.module.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../styles/variables';
+
+.selectButton {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 2.5em;
+  min-height: 2em;
+  cursor:pointer;
+  border-radius: 15%;
+  font-weight: 800;
+  margin-right: 0.5em;
+}
+
+.check {
+  @extend .selectButton;
+  background: $green;
+}
+
+.times {
```

Comment:
I assume this class was labeled `times` because the icon is called that. But since we are using it as an X or close button I think it would make more sense to rename it based on how it is being used

Reply:
renamed to `accept` and `decline` to better indicate how they are used.